src/react/components/pages/homepage/homePage.tsx

```diff
@@ -185,8 +188,11 @@ export default class HomePage extends React.Component<IHomePageProps, IHomePageS
 
         this.props.applicationActions.ensureSecurityToken(project);
 
+        if (project.lastVisitedAssetId !== undefined) {
```

Comment:
Can be simplified to `if(project.lastVisitedAssetId)` since any `null` or `undefined` value will evaluate to false.